config/routes.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  resources :line_items
```

Comment:
resources adds all methods for the controller. is a good practice to just add only the routes for the actions you created.
`resources :line_items, only: [:new, :create]`

same for carts